### ISSUE TITLE
Package as package?

### ISSUE THREAD

**Issue by kenahoo** [helmi77/gc-finance#1]
It would be helpful if this project included a little more packaging so that people could do something like 

```
pip install 'gc-finance @ git+https://github.com/wolfsblu/gc-finance'
```

rather than manually downloading, installing dependencies, and running the scripts.  This would probably involve just creating a `setup.py` or `pyproject.toml`, moving the code files into a module directory, and putting the `main.py` in a `scripts/` directory (probably renaming to something like `gc-finance`).

**Comment by wolfsblu** [comment]
That's a good idea, I will get on it as soon as I find some spare time. Thank you!

**Comment by wolfsblu** [comment]
Solved with 37473ee
